proto/schema.proto

```diff
@@ -0,0 +1,170 @@
+syntax="proto3";
+
```

Comment:
I think we should have a `package` element to clearly distinguish from the CSI spec.  For example, the CSI spec starts with
```
syntax = "proto3";
package csi.v1;
```

Reply:
> Do you think we should have a package element to clearly distinguish from the CSI spec?

I think it's a good idea since the elements are identical to the CSI specs. The clients which are consuming `.proto` files can always update the file if required.

Reply:
Okay, what should it be? `snapshotmetadata` ?

We need to see if it makes the package statements change in the autogenerated `pkg/grpc`. If so, we should rename accordingly.

Reply:
With `snapshotmetadata` package name, the URL path becomes `/snapshotmetadata.SnapshotMetadata/GetMetadataDelta`. But it doesn't change the way exported methods can be used.

Reply:
Does the package name in the `.proto` file get overridden by the value specified in the `protoc` command invocation in the `Makefile`?

Reply:
@carlbraganza that the package path where the generated code is kept. Are you talking about any specific flag?